cumulusci/core/tests/test_config.py

```diff
@@ -341,7 +341,7 @@ def test_repo_url_no_repo_root(self):
 
     def test_repo_url_from_git(self):
         config = BaseProjectConfig(BaseGlobalConfig())
-        self.assertIn("/CumulusCI", config.repo_url)
+        self.assertEquals("git@github.com:SFDO-Tooling/CumulusCI.git", config.repo_url)
```

Comment:
unittest.TestCase.assertEquals is deprecated; use assertEqual (singular) or a pytest assertion instead.